test/parallel_api/algorithm/alg.modifying.operations/transform_binary.pass.cpp

```diff
@@ -118,12 +136,12 @@ main()
     test<std::int32_t, float32_t, float32_t>(non_const(TheOperation<std::int32_t, float32_t, float32_t>(1.5)));
     test<std::int64_t, float64_t, float32_t>(non_const(TheOperation<std::int64_t, float64_t, float32_t>(1.5)));
 #endif
-    // lambda
-    //TODO: wrong value in output sentence with std::int8_t
-    //test<std::int8_t, float64_t, std::int8_t>([](const std::int8_t& x, const float64_t& y) { return std::int8_t(std::int8_t(1.5) + x - y); });
```

Comment:
Did we determine it is not worth pursuing this TODO with `int8_t`?

Reply:
I have just removed the commented code. 
Obviously, we have got integer overflow for our test with "int8" type...  If we want to test such cases we have to write the test more accurately with checking resutl.